src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisoners/PrisonersResource.kt

```diff
@@ -57,12 +57,12 @@ class PrisonersResource(private val prisonerService: PrisonerService) {
   fun getPrisonerIdentifiers(
     @PageableDefault(sort = ["bookingId"], direction = Sort.Direction.ASC)
     pageRequest: Pageable,
-    @RequestParam(value = "active", required = false, defaultValue = "false")
+    @RequestParam(value = "active", required = false, defaultValue = "true")
     @Parameter(
       description = "Only return active prisoners currently in prison",
     )
     active: Boolean = false,
-  ): Page<PrisonerId> = prisonerService.findAllPrisoners(pageRequest, active)
+  ): Page<PrisonerId> = if (active) prisonerService.findAllActivePrisoners(pageRequest) else throw UnsupportedOperationException("Not implemented - only active prisoners can be found")
```

Comment:
my thinking is we will need the other version soon - soon the interface is there but not the implementation for all prisoners